modules/nw-ingress-controller-endpoint-publishing-strategies.adoc

```diff
@@ -5,4 +5,21 @@
 [id="nw-ingress-controller-endpoint-publishing-strategies_{context}"]
 = Ingress controller endpoint publishing strategy
 
+*`NodePort` endpoint publishing strategy*
+
+The `NodePort` endpoint publishing strategy publishes the Ingress Controller using a Kubernetes `NodePort` service.
```

Comment:
This content was pulled from the Ingress Operator API, and may need to change.

Reply:
I don't know if we do this elsewhere, but what about a link to https://kubernetes.io/docs/concepts/services-networking/service/#nodeport?

Reply:
Yeah, we do. I added a link.